sw/device/lib/pinmux.c

```diff
@@ -4,11 +4,9 @@
 
 #include "sw/device/lib/pinmux.h"
 
-#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
-#include "sw/device/lib/common.h"
-
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 #include "pinmux_regs.h"  // Generated.
+#include "sw/device/lib/common.h"
```

Comment:
Nit: Can we try to keep the generated and ungenerated headers separate? I might look at whether I can do this with clang-format going forwards.

Reply:
+1

Reply:
Unfortunately, that wasn't me, that was `clang-format`. I can do the change without running `clang-format` but I'm not sure if we will have any issues in CI. (The version that I have is different and I didn't have a chance to play with docker yet).

Reply:
Ok, no worries, I'll add this to my list to look at today.

Reply:
How about now?

Reply:
Updated the commit, does it look better now?